changelog.txt

```diff
@@ -32,6 +34,7 @@ that repo.
 -@ `gui/unit-syndromes`: allow the window widgets to be interacted with
 -@ `fix/protect-nicks`: now works by setting the historical figure nickname
 -@ `gui/liquids`: fixed issues with unit pathing after adding/removing liquids
+- `makeown`: fixes errors caused by using makeown on an invader
```

Comment:
this.. is not better : P (this is the section for the previous release)

I can fix it up.